### ISSUE TITLE
Some shittos.

### ISSUE THREAD

**Issue by Wickez** [Sumwunn/DoomRPG#100]
1. DoomRLA uniques' messages popping up at the UAC Outpost: what about getting rid of them, please? Or, if they are in the place for those few brave robbers, what about replacing them with a more generic message instead of giving exact information on what has spawned (since it's easy to connect certain messages with certain uniques after a few tries)?

2. DoomRPG and DoomRLA both have the same name for different objects: "demon artifact" (DRLA uses it to improve demonic weapons). Since I tried to convince the creator of DoomRLA to change their object's name to something like "demon relic/vestige", but they didn't feel like it at all, saying it's OK as it is, what about changing it on the DoomRPG's side for more clarity?

3. The messages from the UAC Outpost's pads get bundled together when more than a single pad gets activated. The **Outpost.c** script positions them all in the same spot. What about spacing them apart so the messages wouldn't be all in one spot when that happens?

4. The perk's info for Strength/Defense is too long (I know, because I made them) and crosses the "character sheet" frame:

_"Total damage dealt exponentially increases as HP gets lower"
"Total damage taken exponentially decreases as HP gets lower"_

Replacing the word "exponentially" with "ragingly" makes them fit once again.

**Comment by Someone65** [comment]
I strongly agree with point 2. It confused the hell out of me as a new player before.

Also "ragingly" is just... weird? Maybe just remove the word entirely.

**Comment by Wickez** [comment]
I almost forgot:

5. Is it possible to somehow make DOOM (any) maps appear sequentially in the teleporter (when applied via WadSmoosh) instead the order they were visited first, please? Like, if I entered E1M10 from E1M1, make it appear after E1M9 instead E1M1 (the same goes for all secret maps from DOOM)?

**Comment by Sumwunn** [comment]
All good points.

Note(s):
1. If DRLA does this in ACS it should be an easy fix.
5. I dumped the original sorter (when I redid map code) since it was a bit destructive. I should be able to write a new one easy enough, though.

**Comment by Sumwunn** [comment]
Post to Rebalance GitHub page.

**Comment by Sumwunn** [comment]
Improved 2, 3, and 4 in https://github.com/Sumwunn/DoomRPG/commit/10cdcd7bd876e1715b72c0d3337ce6fc9fe9e78d.
